detox/test/ios/Podfile

```diff
@@ -1,3 +1,5 @@
+ENV['RCT_NEW_ARCH_ENABLED'] = '0'
```

Comment:
The new arch should not be disabled here since the RN 76 core release note activates it. Making something "compatible" with RN 76 would be making it compatible with the New Architecture.

Reply:
Thank you for the feedback! We're gradually implementing support for the new RN. Our first step was ensuring it works without the new architecture. Support for the new architecture is already underway: https://github.com/wix/Detox/issues/4175